feat: add visualiser of documentation of our API

linuxbandit opened the pull request:
One will be able to go to apiadmin.appserver.test and pick on the top-right corner one of our service, and have the docs presented.

Requires use of [swaggerJSDoc](https://github.com/Surnet/swagger-jsdoc) (not a priority)

Imho better than apiary, because the code is auto-generated and in the same place (I used it at work)

linuxbandit committed:
feat: add visualiser of documentation of our API

serge1peshcoff reviewed (approved):
Should work I assume

linuxbandit committed:
fix: addresses for request leaving from browser

it was previously thought that they left from container

linuxbandit committed:
fix: redisadmin wrong environment

the host does not exist yet (and maybe it won't)

the basic auth is done on traefik level, doing it twice causes
problems or something

linuxbandit committed:
feat: upgrade pgadmin

linuxbandit committed:
Merge branch 'master' into feat-api-docs

linuxbandit commented:
@serge1peshcoff do you know why it required me to merge master in here if there was no merge conflict?
